tests/Unit/Infrastructure/ErrorLoggingFileFetcherTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Unit\Infrastructure;
+
+use FileFetcher\FileFetcher;
+use FileFetcher\FileFetchingException;
+use FileFetcher\InMemoryFileFetcher;
+use Psr\Log\LoggerInterface;
+use WMDE\Fundraising\Frontend\Infrastructure\ErrorLoggingFileFetcher;
+use PHPUnit\Framework\TestCase;
+
+class ErrorLoggingFileFetcherTest extends TestCase {
+
+	public function testGivenSucceedingFileFetcher_itsContentIsReturned() {
+		$logger = $this->createMock( LoggerInterface::class );
+		$logger->method( 'error' )->with( $this->never() );
+		$errorLoggingFileFetcher = new ErrorLoggingFileFetcher(
+			new InMemoryFileFetcher( ['song.txt' => 'I\'m a little teapot'] ),
```

Comment:
I'm not sure it was changed... might just have been unspecified in the doc before, since you could not use PHP 5.4 stuff in MediaWiki anyway. AFAIK we used the spacy style in Wikibase since we first started using `[]`. And it's what I've been using in the fun code since the start.

Not sure about the state of the CS. Might be one of those things that got dropped since it was being derp with (nullable) return types. @thiemowmde do you know if there is a sniff for this?